common/lib/dependabot/pull_request_creator/codecommit.rb

```diff
@@ -197,8 +197,7 @@ def pull_requests_for_branch
 
       sig { void }
       def create_commit
-        author = author_details&.slice(:name, :email, :date)
-        author = nil unless author&.any?
```

Comment:
why is line 201 removed?

Reply:
method only expect author_name,  that is T.nilable(String)
removed and added the code get the author name instead `author = author_details&.slice(:name, :email, :date)&.values&.first`

Reply:
If it expects a name, why are do we need the `email` or `date`?

Reply:
This was my suggestion, so let me clarify my thinking, previously we were passing a hash with `name`, `email`, and `string` values, my assumption is that the email and date are meant to be used as backup data when a name isn't available, and with this change we'll use whichever is available in that specific order.

Some more context is that, _we_ don't actually use the `Codecommit` client. It's available for anyone trying to run Dependabot standalone against AWS Codecommit, and I don't think many people are, so at this point I am mostly just happy to silence the Sorbet errors, and anyone actually using the code can contribute a fix if it's not working for them.

Reply:
I'm also fine with `fetch(:name)` 👍